PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -641,6 +654,15 @@ private void CompletionSession_Dismissed(object sender, EventArgs e)
             _activeSession = null;
         }
 
+        private bool? IsInStringArea(bool? isInStringArea)

```

Comment:
The intention is to avoid multiple invoking of the utility method to determine whether it's in string area.